contracts/service-registry/src/state.rs

```diff
@@ -156,29 +156,29 @@ pub enum AuthorizationState {
 
 type ChainNames = HashSet<ChainName>;
 type ServiceName = str;
-type WorkerAddress = Addr;
+type VerifierAddress = Addr;
 
 pub const SERVICES: Map<&ServiceName, Service> = Map::new("services");
-pub const WORKERS_PER_CHAIN: Map<(&ServiceName, &ChainName, &WorkerAddress), ()> =
-    Map::new("workers_per_chain");
-pub const CHAINS_PER_WORKER: Map<(&ServiceName, &WorkerAddress), ChainNames> =
-    Map::new("chains_per_worker");
-pub const WORKERS: Map<(&ServiceName, &WorkerAddress), Worker> = Map::new("workers");
+pub const VERIFIERS_PER_CHAIN: Map<(&ServiceName, &ChainName, &VerifierAddress), ()> =
+    Map::new("verifiers_per_chain");
+pub const CHAINS_PER_VERIFIER: Map<(&ServiceName, &VerifierAddress), ChainNames> =
+    Map::new("chains_per_verifier");
+pub const VERIFIERS: Map<(&ServiceName, &VerifierAddress), Verifier> = Map::new("verifiers");
```

Comment:
these sorts of changes require a migration right?

Reply:
ya, all of them need migration. will do in separate pr